docs/sources/administration/api-keys/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: 'API keys in Grafana'
+menuTitle: 'API keys'
+description: 'This contains information about API keys in Grafana'
+aliases: [docs/sources/administration/api-keys/_index.md]
```

Comment:
We're working to better understand if adding aliases before a topic actually moves is something we should do. Docs squad is in active dialog about when to add them. In addition, I've used incorrect paths for aliases in my work. The prefix should be "docs/grafana" and not "docs/sources". 

I think you can safely remove all aliases until we have more solid guidance that I can share with you.

Reply:
aliases removed 👍

Reply:
will not resolve this as this is important for future doc implementations